source/documentation/common/install/proc-Configuring_the_Red_Hat_Virtualization_Manager.adoc

```diff
@@ -281,7 +281,7 @@ Confirm engine admin password:
 ----
 Application mode (Both, Virt, Gluster) [Both]:
 ----
-*Both* offers the greatest flexibility. In most cases, select `Both`. *Virt* allows you to run virtual machines in the environment; *Gluster* only allows you to manage GlusterFS from the Administration Portal.
+*Both* - offers the greatest flexibility. In most cases, select `Both`. *Virt* - allows you to run virtual machines in the environment; *Gluster* - only allows you to manage GlusterFS from the Administration Portal.
```

Comment:
1. I don't think hyphen are the best choice. Consider one of the following possibilities:
a. no punctuation. The sentences read clearly without it.
b: a colon
c: rewrite as a bulleted list
2. re: "In most cases, select `Both`. I would *Bold* ""Both," unless the user actually types it -- in that case, i'd use "type" as the verb, not select.

Reply:
I found the wording slightly annoying as is - "Both offers the..." looked like a typo at first, so I added the hyphens to make it unambiguous.  Used your suggestion for bulleted list...